### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
I'm not sure what your goals are for this repository.

Since you have a historical branch, typos can safely be left there...

I'm also not sure which functions are APIs and thus can't be renamed.

I've dropped one prototype since afaict it's unused.

There's at least one end-user facing string change, which I'd hope you'd want to take.

### CODE REVIEW

File: src/res.rc [hunk 1]
```diff
@@ -631,7 +631,7 @@ BEGIN
     MH_MYITEMS+IDM_BYTYPE,      "Sorts files by type"
     MH_MYITEMS+IDM_BYSIZE,      "Sorts files by size"
     MH_MYITEMS+IDM_BYDATE,      "Sorts files by date"
-    MH_MYITEMS+IDM_BYFDATE,     "Sorts files by foward date"
+    MH_MYITEMS+IDM_BYFDATE,     "Sorts files by forward date"
```
Comment (jsoref) [inline src/res.rc]: End-user facing

File: src/spinlock.h [hunk 2]
```diff
@@ -51,7 +51,7 @@ void SpinLock::Lock()
 		/*
 		Spinning in a loop of interlockedxxx calls can reduce the available memory bandwidth and slow
 		down the rest of the system. Interlocked calls are expensive in their use of the system memory
-		bus. It is better to see if the 'dest' value is what it is expected and then retry interlockedxx.
+		bus. It is better to see if the 'dest' value is what it is expected and then retry interlockedxxx.
```
Comment (jsoref) [inline src/spinlock.h]: This is for consistency with two lines earlier...

File: src/treectl.c [hunk 3]
```diff
@@ -2704,8 +2704,8 @@ TreeControlWndProc(
 
       UpdateWindow(hwndStatus);
 
-      iSelHilite = ((LPDROPSTRUCT)lParam)->dwControlData;
-      RectTreeItem(hwndLB, iSelHilite, (BOOL)wParam);
+      iSelHighlight = ((LPDROPSTRUCT)lParam)->dwControlData;
```
Comment (jsoref) [inline src/treectl.c]: I recognize that `Hilite` is used as an alternate spelling of `Highlight`, but `Hilight` is definitely not, so I'm suggesting standardizing on the long form (which also exists in the repo).

File: src/wfassoc.c [hunk 4]
```diff
@@ -144,7 +144,6 @@ PEXT pExtBase = NULL;
 
 INT APIENTRY AssociateFileDlgProc(register HWND hDlg, UINT wMsg, WPARAM wParam, LPARAM lParam);
 BOOL AssociateDlgInit(HWND hDlg, LPTSTR lpszExt, INT iSel);
-VOID AssoicateFileDlgCommand(HWND hDlg, WPARAM wParam, LPARAM lParam, PASSOCIATEFILEDLGINFO pAssociateFileDlgInfo);
```
Comment (jsoref) [inline src/wfassoc.c]: This is a misspelling that isn't used anywhere, the correct spelling has a different prototype below (`BOOL`)

File: src/wfgoto.cpp [hunk 5]
```diff
@@ -703,7 +703,7 @@ BuildDirectoryTreeBagOValues(PVOID pv)
 }
 
 DWORD
-StartBuildingDirectoryTrie()
+StartBuildingDirectoryTree()
```
Comment (jsoref) [inline src/wfgoto.cpp]: `Trie` is a CS term, but afaict this isn't that kind of object.

File: src/wfutil.c [hunk 6]
```diff
@@ -1215,14 +1215,14 @@ IsWild(LPTSTR lpszPath)
 
 /*--------------------------------------------------------------------------*/
 /*                                                                          */
-/*  CheckSlashies() -                                                       */
+/*  CheckSlashes() -                                                        */
```
Comment (jsoref) [inline src/wfutil.c]: There could be a reason for `Slashies`, but I don't see it.

File: src/wfcopy.c [hunk 7]
```diff
@@ -1365,7 +1365,7 @@ GetNameDialog(DWORD dwOp, LPTSTR pFrom, LPTSTR pTo)
 //           FUNC_COPY   - Copy files in pFrom to pTo
 //
 // OUTC pdwError         -- If error, this holds the err code else 0
-// INC  bIsLFNDestFrive  -- Is the dest drive lfn?
+// INC  bIsLFNDriveDest  -- Is the dest drive lfn?
```
Comment (jsoref) [inline src/wfcopy.c]: See line 1398...

**Comment by craigwi** [comment]
I'm really impressed that folks care so much about such details.  I'm good with that!

File: src/treectl.c (hunk 3 above)
Comment (ZanderBrown) [reply src/treectl.c]: Hilight is a logical shortening of Highlight (aka Hi~~gh~~light or hi-light)

**Comment by jsoref** [comment]
#48 for Trie

**Pull request merged**